frontend/src/App.vue

```diff
@@ -116,14 +154,21 @@
     <div v-if="showCookieMsg" id="cookies" class="has-background-grey">
       <div class="column has-text-centered p-1">
         <div class="has-text-white">
-          We use cookies to enhance the usability of our website.
-          By continuing you are agreeing to our
-          <router-link class="has-text-white has-text-weight-bold"
-                       :to="{ name: 'about', hash: '#privacy' }">
-            Privacy Notice and Terms of Use
-          </router-link>&emsp;
-          <p class="button is-small is-rounded has-background-danger has-text-white has-text-weight-bold"
-             @click="showCookieMsg=false; acceptCookiePolicy()">
+          We use cookies to enhance the usability of our website. By continuing you are agreeing to
+          our
+          <router-link
+            class="has-text-white has-text-weight-bold"
+            :to="{ name: 'about', hash: '#privacy' }"
+          >
+            Privacy Notice and Terms of Use </router-link
+          >&emsp;
+          <p
+            class="button is-small is-rounded has-background-danger has-text-white has-text-weight-bold"
+            @click="
+              showCookieMsg = false
```

Comment:
Can somebody confirm that no semicolon is needed here?

Reply:
I can confirm that ';' is not needed here.